docs/guides/headless-cms/content-translations.md

```diff
@@ -0,0 +1,155 @@
+
+![A Directus item detail page is shown using the Translations interface. Editable fields for the English and French translations for an article are displayed side-by-side. Fields include Title, Slug, Summary, and Content. ](https://marketing.directus.app/assets/64eee60e-605b-4d2c-809f-83557ed0d5ef.png?key=doc)
+
+## Before You Start
+
+You will need:
+
+- A Directus project. The easiest way to get started with Directus is with our
+  [managed Directus Cloud service](https://directus.cloud). You can also self-host Directus.
+- Familiarity with [creating collections](/app/data-model/collections#create-a-collection) and
+  [creating fields](/app/data-model/fields#create-a-field-standard) inside Directus.
+
+## Create Languages Collection
+
+First, you'll need to create a new collection for all the different languages you'll support.
+
+1. Go to Settings > Data Model and create a new collection for `languages`.
+
+   Here's an overview of the schema for this collection.
+
+   ```md
+   languages
+
+   - code (Primary Key Field, Type: Manually entered string )
+   - name (Type: String, Inferface: Input)
+   - direction (Type: String, Interface: Dropdown, Default Value: ltr)
+   ```
+
+   a. When creating the collection, set the primary key to `code` and use the `Manually entered string` type. There's no
+   need to add Optional System Fields.
+
+   ![The Creating New Collection form overlay is shown. The active tab is Collection Setup. Name, Singleton, Primary Key Field, and Type fields are shown and editable by the user.](https://marketing.directus.app/assets/e4fe9a63-9cfd-48d9-a1a9-7aedff6279a9.png?key=doc)
+
+   b. Add another field for the `name` of the language.
+
+   c. Add one last field for the `direction` of the text if you're supporting languages that read right to left.
+
+Now you're ready to move on.
+
+## Create Content Collection
+
+Before you can create translations, you need a collection of content to translate.
+
+2. Create a new collection for `articles`.
+
+   Here's an overview of the initial schema for this collection.
+
+   ```md
+   articles
+
+   - id (Primary Key Field, Type: uuid )
+   - status (Type: String, Interface: Dropdown, Options: ['published','draft','archived'])
+   - image (Type: Image)
+   - author (Type: M2O Relation, Related Collection: directus_users)
+   - date_published (Type: Timestamp, Interface: Datetime)
+   ```
+
+   ![Data Model settings screen for the Articles collection is displayed. The following fields are shown: id, status, date_published, author, image.](https://marketing.directus.app/assets/fdeb07d9-d11a-42e9-a39a-14f0a196ecc6.png?key=doc)
+
+   Create the fields above inside the collection.
+
+   They will be shared across all the different translations of each article.
+
+   Do not add the fields to be translated like `title` or `content` or `slug` yet, because those will be stored in a
+   seperate collection you will create next.
```

Comment:
```suggestion
   Do not add the fields to be translated like `title` or `content` or `slug` yet, because those will be stored in a separate collection shown below.
```